core/common/src/Instant.kt

```diff
@@ -381,6 +384,7 @@ public fun Instant.plus(unit: DateTimeUnit.TimeBased): Instant =
  *
  * The return value is clamped to the platform-specific boundaries for [Instant] if the result exceeds them.
  */
+@Deprecated("Use the minus overload with explicit amount of units", ReplaceWith("this.minus(1, unit)"))
 public fun Instant.minus(unit: DateTimeUnit.TimeBased): Instant =
     plus(-1L, unit)
```

Comment:
I like the practice we have in coroutines: when deprecating something, we move it either to the bottom of the file or altogether to a different file designated for deprecated things. This way, the deprecated (and no longer all that relevant) things are out of sight. Maybe we could adopt it here as well?

Reply:
We'll clean up these deprecations near the stabilization, so it's not relevant here. We can consider this practice after stabilization, however I'm not currently fond of it because it makes history digging with git blame harder.